examples/gcp_cli.py

```diff
@@ -96,7 +96,18 @@ def QueryLogs(args: 'argparse.Namespace') -> None:
     args (argparse.Namespace): Arguments from ArgumentParser.
   """
   logs = gcp_log.GoogleCloudLog(args.project)
-  results = logs.ExecuteQuery(args.filter)
+
+  qfilter = ''
+  if args.start:
```

Comment:
It might be nice to check that `--start` and `--end` are properly formatted and fail early if not.

Reply:
Done.

Reply:
If you're just gonna catch and raise the same error, maybe it would be worth displaying a message to the user explaining the error?